m4/plan.sh

```diff
@@ -16,8 +16,6 @@ pkg_deps=(
   core/glibc
 )
 pkg_build_deps=(
-  core/coreutils
-  core/diffutils
```

Comment:
`diffutils` looks to be necessary for `do_check`.  I don't think we want to include it in build deps though since it isn't actually needed for a normal build.  I think this is fine to leave out for now but we'll need to look at workarounds for the refresh and start to think about solutions longer term.